ow_plexil/src/plans/TestPanTilt.plp

```diff
@@ -0,0 +1,21 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// This is a simplified and stubbed version of a procedure that should
+// "interrogate" the terrain to find a good sampling location.  Instead, we
+// choose an arbitrary location and probe for ground position there.
```

Comment:
Old comment.  I think it would be good to make this a more rigorous test procedure for the antenna.  Have it pan/tilt to a variety of interesting locations that exercise the timeout.

Reply:
Sounds good, I just pushed changes that add more tests. I also found during testing that the joint angles are continuous but the "reached" check assumes that it is bounded between -360 to 360 so I had to add some additional checks. Thanks!